caikit/config/config.py

```diff
@@ -133,13 +134,12 @@ def merge_configs(
             The base config that will be updated with the overrides
         overrides: Optional[dict]
             The override config
-        merge_strategy: str
-            The merging strategy, either `merge` or `override`
+        merge_strategy (str): The merging strategy, either `merge` or `override`
             `override` will replace values in base with those from overrides
             `merge` will deep-merge dictionaries and prepend-merge lists
     Returns:
-        merged: dict
-            The merged results of overrides merged onto base
+			merged: dict: The merged results of overrides merged onto base
```

Comment:
These are tricky: the name of the thing to be returned (`merged` here) should not be in the docstring and we'd probably want to go ahead and delete it. If you can find a good enough heuristic to catch most of these cases we can probably clean up what's left